### PR TITLE
www/caddy: Added lang() function for all user exposed text for translation

### PR DESCRIPTION
**Pull request by Monviech**
Added lang() function for all user exposed text for translation, I hope I was thorough enough with it.

Of course there is still some text that won't be translated, like some error messages that are returned from the backend (for example the Caddyfile validation messages, they're always english.)

### CODE REVIEW

File: www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/general.volt [hunk 1]
```diff
@@ -111,12 +111,12 @@
                     // Save the form before continuing
                     saveFormToEndpoint("/api/caddy/general/set", 'frm_GeneralSettings',
                         function() {  // callback_ok: What to do when save is successful
-                            showAlert("Configuration saved successfully. Please don't forget to apply the configuration.", "Save Successful");
+                            showAlert("{{ lang._('Configuration saved successfully. Please do not forget to apply the configuration.') }}", "{{ lang._('Save Success') }}");
```
Comment (fichtner) [inline www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/general.volt:114]: ah the old contractions-are-not-for-technical-writing is enforced here due to the weird quoting rules of volt strings ;)

**Comment by fichtner** [comment]
For error messages from caddy we can't do much. That's ok. Controller texts are translated using gettext() wrapper in the same manner if not done already.

I'm merging this. Make sure to double-check the JS as the escaping in there is extra-finicky... Thanks!

File: www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/general.volt [hunk 2]
```diff
@@ -134,12 +134,13 @@
...
-    <li><a data-toggle="tab" href="#dnsProviderTab">DNS Provider</a></li>
-    <li><a data-toggle="tab" href="#dynamicDnsTab">Dynamic DNS</a></li>
-    <li><a data-toggle="tab" href="#logSettingsTab">Log Settings</a></li>
+    <li class="active"><a data-toggle="tab" href="#generalTab">{{ lang._('General') }}</a></li>
+    <li><a data-toggle="tab" href="#dnsProviderTab">{{ lang._('DNS Provider') }}</a></li>
+    <li><a data-toggle="tab" href="#dynamicDnsTab">{{ lang._('Dynamic DNS') }}</a></li>
+    <li><a data-toggle="tab" href="#logSettingsTab">{{ lang._('Log Settings') }}</a></li>
 </ul>
 
+
```
[the remark this answers is not in the document]
Comment (Monviech) [reply www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/general.volt]: I misses that xD

File: www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/general.volt (hunk 1 above)
Comment (Monviech) [reply www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/general.volt:114]: I found out the hard way. I will avoid them.

**Pull request merged**